modules/rules_build_error/0.7.0/presubmit.yml

```diff
@@ -0,0 +1,20 @@
+---
+bcr_test_module:
+  module_path: "examples"
+  matrix:
+    platform:
+      - "debian10"
+      - "ubuntu2004"
+      - "macos"
```

Comment:
can you change this to `macos_arm64` and see if it helps with the CI failure?

Reply:
Oh the CI has succeeded